Update README.md (#15)
vtex-apps/order-shipping: docs/README.md

> 📢 Use this project, [contribute](https://github.com/{OrganizationName}/{AppName}) to it or open issues to help evolve it using [Store Discussion](https://github.com/vtex-apps/store-discussion).

Reviewer: ```suggestion
📢 Use this project, [contribute](https://github.com/vtex-apps/order-shipping) to it or open issues to help evolve it using [Store Discussion](https://github.com/vtex-apps/store-discussion).
```

> # ORDERS SHIPPING

Reviewer: ```suggestion
# Order Shipping
```

> ⚠️ **This is an ongoing, unsupported, unfinished and undocumented project. We do not guarantee any results after installation.** ⚠️

Reviewer: ```suggestion
> :warning: *Although ready to be installed, the Order Shipping app is currently an **open-beta project**. Bear in mind that due to that status, you can expect an accelerated state of evolution.*
```

<!-- DOCS-IGNORE:start -->
<!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->
[![All Contributors](https://img.shields.io/badge/all_contributors-0-orange.svg?style=flat-square)](#contributors-)
<!-- ALL-CONTRIBUTORS-BADGE:END -->
<!-- DOCS-IGNORE:end -->

> Centralizes all shipping related requests to the Checkout API.

> Any kind of shipping query or manipulation should be made through this component. This ensures that each interaction with the Checkout API happens in succession, avoiding concurrency issues.

Reviewer: ```suggestion
The Order Shipping app manages all shipping queries, ensuring that each interaction with the [Checkout API](https://developers.vtex.com/vtex-developer-docs/reference/catalog-api-overview) happens in succession i.e. with no concurrency issues. 
```

```tsx
selectDeliveryOption('PAC')

> ```

Reviewer: I believe this info should be divided into different sections: Configuration and Modus Operandi. The `Configuration` section should teach users how to set up the app in their VTEX accounts and/or their store themes. In turn, the `Modus Operandi` section should contain further info regarding the app behavior i.e. how users should properly use it.

Author: The app is still under construction, and most probably it will become private.. It is really urgent to update the readme with the warning for now 🙏

## Customization

> TBC

Reviewer: ```suggestion
🚧 To be defined.
```
